docs/language/api.rst

```diff
@@ -850,16 +850,26 @@ eval-when-compile
 -----------------
 
 
-first / car
------------
+first
+-----
 
-``first`` and ``car`` are macros for accessing the first element of a collection:
+``first`` is a function for accessing the first element of a collection.
 
 .. code-block:: clj
 
     => (first (range 10))
     0
 
+It is implemented as ``(next (iter coll) None)``, so it works with any
+iterable, and if given an empty iterable, and it will return ``None`` instead
```

Comment:
Second *and* is redundant (though I feel like this might be easier to read if the section on empty literals was a separate sentence).